kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -329,6 +331,10 @@ def kube_pod_status_phase(self, message, **kwargs):
                                          tags=[self._label_to_tag("pod", metric.label),
                                                self._label_to_tag("namespace", metric.label)] + self.custom_tags)
 
+            tags = [self._label_to_tag("pod", metric.label), self._label_to_tag("namespace", metric.label),
+                    self._label_to_tag("phase", metric.label)] + self.custom_tags
```

Comment:
I'm nit-picking but I think we can just combine all the tags into a single variable and provide that to both `self._condition_to_tag_check(metrics, ...)` and `self.gauge(metric_name, ...)` calls. This will also add the phase as a tag to the service checks, which may provide some extra information.